CMakeLists.txt

```diff
@@ -11,16 +11,11 @@ if (POLICY CMP0042)
 endif()
 
 set(CMAKE_USER_MAKE_RULES_OVERRIDE_CXX "${CMAKE_CURRENT_SOURCE_DIR}/cmake/cxx_compiler_flags_overrides.cmake")
-project(Z3 CXX)
+project(Z3 VERSION 4.8.6.0 LANGUAGES CXX)
 
 ################################################################################
 # Project version
 ################################################################################
-set(Z3_VERSION_MAJOR 4)
```

Comment:
If the version is provided to the ``project`` command (possibly since cmake 3.0), then these are defined automatically.

Reply:
thanks for heads up. Seems nice to have, but will pass on updating right away (because I could be doing it wrong).